types/tabulator-tables/index.d.ts

```diff
@@ -1376,6 +1376,7 @@ declare class Tabulator {
     browserSlow: boolean;
     modules: any;
     options: Tabulator.Options;
+    element: any;
```

Comment:
element is not part of the Tabulator class. It is a parameter that can be passed in but not saved to this

 `var Tabulator = function(element, options){`

Reply:
Thanks for all your replies.
We did this particular edit because we have to get the id of a tabulator javascript object. 
Using table.element.id this can be done, but if element is not declared we can't use it. 
table.element returns the html object of the table.
Do you know how to get the id?
Thanks

Reply:
From https://github.com/olifolkerd/tabulator/blob/master/src/js/core.js :
```
Tabulator.prototype.initializeElement = function(element){
	if(element instanceof HTMLElement){
		**this.element** = element;
```

I think **this.element** is part of tabulator class.

Reply:
I gotcha now. I'm good with it.